index.html

```diff
@@ -0,0 +1,198 @@
+    </div>
+    <div class="main-date">
+      <h3>2022.10.15(THU)~16(FRI)</h3>
+      <p>@Governemnt Secondry School Owerri</p>
+    </div>
+  </div>
+  </section>
+
+  <section class="main-program">
+    <div class="main-program-title">
+      <h2>Main Program</h2>
+      <hr class="main-hr">
+    </div>
+    <div class="main-program-card-container">
+    <div class="main-program-card">
+      <div>
+        <i class="fa-solid fa-award main-program-image"></i> 
+      </div>
+      <div class="main-program-card-title">
+        <h4>Award</h4>
+      </div>
+      <div class="main-program-card-description">
+        <p>
+          Lorem ipsum dolor sit, amet consectetur
+          adipisicing elit. Nostrum minima.
+        </p>
+      </div>
+    </div>
+
+    <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-place-of-worship main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Symposia</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-people-arrows main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Reachout</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-plane-arrival main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Home-coming</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-business-time main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Business talk</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+    </div>
+    <p class="see-whole-program"><a href="#"> SEE THE WHOLE PROGRAM</a></p>
+    <div class="main-program-card-btn"><button class="join-btn" type="button">Join GSSO Alumni Club</button></div>
+  </section>
+  <section class="featured-speakers">
+    <div class="featured-speakers-title">
+      <h1>Featured Speakers</h1>
```

Comment:
Kindly replace this `h1` tag with `h2` since there should only be one `h1` tag per page following the best practices 🤗